d2l-typography-shared-styles.html

```diff
@@ -3,7 +3,8 @@
 <dom-module id="d2l-typography-shared-styles">
```

Comment:
@dbatiste As we discussed out-of-channel, you're correct that wrapping these mixins in a dom-module isn't necessary, so I've removed it. The `include` is no longer necessary, but `is="custom-style"` is needed for non-web-component scenarios.